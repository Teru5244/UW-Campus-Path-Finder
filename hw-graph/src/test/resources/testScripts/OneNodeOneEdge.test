# Create a graph
CreateGraph g

# Add one node
AddNode g n

# Add one edge
AddEdge g n n e

# Print the nodes in the graph and the outgoing edges from n
ListNodes g
ListChildren g n