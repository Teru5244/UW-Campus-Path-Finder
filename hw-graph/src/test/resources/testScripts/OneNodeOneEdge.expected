# Create a graph
created graph g

# Add one node
added node n to g

# Add one edge
added edge e from n to n in g

# Print the nodes in the graph and the outgoing edges from n
g contains: n
the children of n in g are: n(e)